interface/super/edit_layout.php

```diff
@@ -1562,26 +1564,37 @@ function myChangeCheck() {
             }
             $gmyname = $grparr[$group_id]['grp_title'];
 
-            echo text($gdispname);
-            // if not english and set to translate layout labels, then show the translation of group name
-            if ($GLOBALS['translate_layout'] && $_SESSION['language_choice'] > 1) {
-                // echo "<span class='translation'&gt;&gt;&gt;&nbsp; " . xlt($gdispname) . "</span>";
-                echo "<span class='translation'>" . xlt($gdispname) . "</span>";
-                echo "&nbsp; ";
-            }
-            echo "&nbsp; ";
-            echo " <input type='button' class='addfield' id='" . attr("addto~$group_id") . "' value='" . xla('Add Field') . "'/>";
-            echo "&nbsp; &nbsp; ";
-            echo " <input type='button' class='renamegroup' id='" . attr("$group_id~$gmyname") . "' value='" . xla('Rename Group') . "'/>";
-            echo "&nbsp; &nbsp; ";
-            echo " <input type='button' class='deletegroup' id='" . attr("$group_id") . "' value='" . xla('Delete Group') . "'/>";
-            echo "&nbsp; &nbsp; ";
-            echo " <input type='button' class='movegroup' id='" . attr("$group_id~up") . "' value='" . xla('Move Up') . "'/>";
-            echo "&nbsp; &nbsp; ";
-            echo " <input type='button' class='movegroup' id='" . attr("$group_id~down") . "' value='" . xla('Move Down') . "'/>";
-            echo "&nbsp; &nbsp; ";
-            echo "<input type='button' value='" . xla('Group Properties') . "' onclick='edit_layout_props(" . attr_js($group_id) . ")' />";
-            echo "</div>";
+            $group_id_attr = attr($group_id);
```

Comment:
prob need a $group_id_attr_js variable for when using it in edit_layout_props() line below